web/sites/all/modules/onlinekurslabor/nm_stream/nm_stream.module

```diff
@@ -2967,7 +2968,19 @@ function nm_stream_node_get_poll_json($node) {
     }
   }
 
+  // check end date
+  if(!empty($poll->endDate)) {
+    $endDate = strtotime($poll->endDate);
+    // check if poll is closed
+    if(time() > $endDate) {
+      $final_results = TRUE;
```

Comment:
das is eigentlich inkorrekt. Du müsstest hier die drupal-timeshift-funktionen mit einbauen.
sonst kann der philippino nicht mehr teilnehmen, obwohl die umfrage noch offen wäre.

Aber fraglich, ob das relevant ist...

Reply:
ich gehe davon aus, dass strtotime die TimeZone des aktiven Benutzers akzeptiert und das dadurch passt... 
habe einen kleinen unit test geschrieben:
https://github.com/n-multimedia/onlinekurslabor/commit/c6e4318a336951f98413fa475d75f0cb6caa5dc9

Reply:
oh. mir war nur, dass ich da mal was umgestzt hatte mit dem problem. betraf aber wohl eher wordpress...
Danke, jetzt wissmas 👍